hardhat.config.ts

```diff
@@ -4,7 +4,7 @@
 module.exports = {
   solidity: '0.8.7',
   networks: {
-    hardhat: { chainId: 1337 },
```

Comment:
If I may suggest, it would be better to have this `hardhatChainId` calculated outside.